### PR TITLE
add sample status yaml file

### PR DESCRIPTION
**Pull request by mauerbac**
@tscanlin CC: @krishnan-chandra

### CODE REVIEW

**Comment by tscanlin** [comment]
LGTM

File: src/content/data/export/index.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ The file name follows the format: `experiment_id-yyyy-mm-dd.tsv.gz` . Example: `
 
 ### Status File
 
-A status file will be provided to track the success or failure of that day's experiment event files. This file is named `status.yaml` and is included in the daily folder per project. The format contains: `failed_exports`, `successful_exports`, and a timestamp in UTC seconds since epoch.
+A status file will be provided to track the success or failure of that day's experiment event files. This file is named `status.yaml` and is included in the daily folder per project. The format contains: `failed_exports`, `successful_exports`, and a timestamp in UTC seconds since epoch. View a <b>sample YAML <a href="/data/sample-status.yaml">file</a>.</b>
```
Comment (mauerbac) [inline src/content/data/export/index.md]: hey @tscanlin, the sample yaml file link worked locally, but seems broken in production -- any idea? The file has been committed, as you can see below.

Comment (tscanlin) [inline src/content/data/export/index.md]: You could probably change this too: `<a href="../sample-status.yaml">file</a>`

That will go up one directory and look for the file.

Comment (mauerbac) [inline src/content/data/export/index.md]: Cool, thanks! Looks like it's working now.

**Pull request merged**